_posts/2021-02-03-new-ansible-module-copr.md

```diff
@@ -0,0 +1,54 @@
+---
+title: New ansible module copr
+author: Silvie Chlupova
+layout: post
+---
+
+Our new ansible module copr has also reached the last release of the [community general collection](https://github.com/ansible-collections/community.general/releases/tag/2.0.0). Due to the fact that many people are used to using ansible modules to quickly automate their tasks, in the past there has been a requirement to add a copr module to ansible. However, the ansible core team no longer accepts new modules and rather supports users in adding new modules to collections in the ansible galaxy. We didn't want to add our code for the Copr module to one of the existing modules, so we decided to create our own module, which was added to the community general collection in [ansible galaxy](https://galaxy.ansible.com/community/general). The module code can be found in the github repository of the mentioned [collection](https://github.com/ansible-collections/community.general/blob/2.0.0/plugins/modules/packaging/os/copr.py).
+
+## Synopsis
+
+This module can enable, disable or remove the specified repository.
+
+## Quick setup
+
+To install a collection hosted in Galaxy:
+
+    ansible-galaxy collection install community.general
+
+You can also directly use the tarball (version 2.0.0):
+
+    ansible-galaxy collection install https://github.com/ansible-collections/community.general/archive/2.0.0.tar.gz -p ./collections
```

Comment:
It may be worth noting that you can `dnf install ansible-collection-community-general` once that package will be updated to 2.0. Right now it is 1.0.